build.rs

```diff
@@ -136,7 +137,34 @@ fn build_ossl() {
         .expect("Couldn't write bindings!");
 }
 
+#[cfg(feature = "dynamic")]
```

Comment:
should we do the CI target with dynamic linking to make sure this keeps working?

Reply:
good idea!